.github/workflows/ci.yml

```diff
@@ -247,7 +248,9 @@ jobs:
       - name: LLVM instrument coverage
         id: cov
         uses: ./.github/actions/cargo-llvm-cov
-        if: steps.prepare-meta.outputs.has-image
+        # instrument coverage breaks the zig image tests, which is
```

Comment:
It seems the issue was the LLVM code coverage, which caused the internal toolchain to fail.

Reply:
comment is unneccessary, can be removed